static/templates/formatting_buttons_popover.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="formatting_container">
+    <span class="formatting_button">
+        <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAABmJLR0QA/wD/AP+gvaeTAAABSklEQVRIibXVSytFURjG8d9BGLoUpUjKTBmYMJMBn8BApjIxNfUxlHwBM8lAqfMRGDBCIQNJ5DKQS26Dc06ObS3nsvd5atVe7356/+tda72tnB/14dL/esYTznGIXewUYxXVh686xi2W0dwoQGlso6WRgC+sJJPmEoDQGawjX/zuwAjm0R3w3hTzvNdSwWLA24mjiH+03NgUIlWhe2xG/vVkAYDHSPw6C0AO04H4HU7qBeTQi0lsYSrgWcNLLEHaa7qD9v9WmAawqkKTZVHBqcL2/VI1jbaPs7J5N4bRH/A+YwIHtVQQajSYw0fAH+uPmgGwEfC/o61kSNNoFLYvqWYMZgXoisTfsgB0KJxDUq+4Kk0q3l2MYbZs3oohLGAg4M+LdHMWD84HxmMrTQv4xFIseVrAMWZCSas5g6Re8IAL7Ck89vliBX/0DWk+1xOxmky2AAAAAElFTkSuQmCC" data-format="bold"/>
+    </span>
+    <span class="formatting_button">
+        <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABoAAAAaCAYAAACpSkzOAAAABmJLR0QA/wD/AP+gvaeTAAAA60lEQVRIieWVuw5BQRRFl0dDJyEKjUoiEgWNTpRq0Sip+Az/5dXwARoVFYkEiYLOoyKYI+a4k1uwk2kmM3vNrLnJhX9NDbgoxuK1IGgJKisPNlSuvycCFIAucMK8wQZoATkg9i3kNRMB1HFV/piVAMq4hmQFyNJmo+3HcEtFmOv5BRooOz4mAKwx1aVdg/ICZG67WaNO0tb3C+T8fULADlNdyjWoKEBmmgJbdZ61+QaySRjY86ztDCRdg0qY7zPVltioqwtzxh/US+JAGzhi3mgLNICEF0ATOAjl78bIplRSVwWiioONFWt/KFeQ210Hh0ikTAAAAABJRU5ErkJggg==" data-format="italic"/>
+    </span>
+    <span class="formatting_button">
+        <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABoAAAAaCAYAAACpSkzOAAAABmJLR0QA/wD/AP+gvaeTAAAB30lEQVRIia3Wv2sUQRjG8c/dRUQbRawVogYUjHp97LUQFMTO/AliGXKEJApaKFgKEkgsxMYfWNgIFoJFRKNRLGyvE5MyYHGixdyR2WF3cuvlhYHded95v/vOPvPuNoxuLVzFZUygh3U8xrtdyE8/8Vf8rRgr2DsqpI3NDGQwlqExAuQNDkVzH/FSqGAaRyLf+f+FpJXcEd7VwA6jG/kf7QZkviJ2MYpZbdaAHMAVYXu+9OcWMqA/0XWrImYoO5XxjWHNdkVPxoZI2MBFXBLk/Fs4JyuZNbdxLrp/sRPkGD4ol22vnzDd/rkkbq0kpmBtbFRA4nE9WtNJfBs4vhMkVdc67glS3Yrmb/TXzCbxm4rbNxRkUVE548KJv499mCmp5GxdyAz244EggPEovoVbSfwvnKkLme37bkZzW1jqV/OtBDKZg0yUQDqRfzrxlY2fOJ2DtIQXHS+aS2KagpR7FZD3OJqDwLVk0d1M7Emhgb7GWzzEBUN+BZ5GkE/yfSnXdrLWxIno/pViM4xtHt+FhrokVH6wDmzVdkULGUh6GNt1IISTPkjQFT5aA2sJTz4yBKaSRF3hEHYUW/1IkIEtK5ftrkIIPxQ52GdF0dS2VP9TQheYxB78wHM8U63GoewfAo7YbZrQf6sAAAAASUVORK5CYII=" data-format="link"/>
```

Comment:
What's the thinking behind having tiny PNGs here rather than using icons?  (Does Font Awesome not have the icons we need?  We also could think about moving these to our `static/assets/icons` icon font).

Reply:
Font awesome doesn't have good icons for these in v3.  Using icons leads to link icons appearing weird 
![image](https://user-images.githubusercontent.com/25124304/132333534-31305600-88e7-4d43-bbd6-e20493429826.png)

I have tried several.

Reply:
OK -- we should probably just move these icons to live in a system intended to store images/icons, not inline in the HTML, then.